Source/ZoomNet/Resources/CloudRecordings.cs

```diff
@@ -372,6 +367,25 @@ public Task<Stream> DownloadFileAsync(string downloadUrl, CancellationToken canc
 				.AsStream();
 		}
 
+		/// <inheritdoc/>
+		public async Task<Stream> DownloadFileWithouBufferingAsync(string downloadUrl, CancellationToken cancellationToken)
```

Comment:
Can you think of a downside to downloading a file without buffering? I can't think of one. Which leads me to think that all files, large or small, should probably be downloaded without buffering. Which means there's no need for a separate `DownloadFileWithouBufferingAsync` method and you can move the logic to the `DownloadFileAsync` method.

Reply:
From this point of view, then you are right =) 
I have removed the method separating.